integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -279,12 +285,30 @@ public int getattr(String path, FileStat stat) {
       stat.st_mtim.tv_sec.set(ctime_sec);
       stat.st_mtim.tv_nsec.set(ctime_nsec);
 
-      // for shell-based group mapping, use the uid and gid of the corresponding user registered in
-      // unix; otherwise use uid and gid of the user running alluxio-fuse
-      if (mIsShellGroupMapping) {
+      /**
+       * User group rule:
+       *
+       * A) Shell user group translation service disabled. The user and group for all FUSE files
```

Comment:
can we inline the interesting parts of this? Right now we've basically written the code twice, once in Java and once in English

Reply:
Inline those comments to make it cleaner! Thanks
```
      if (mIsUserGroupTranslation) {
        // Translate the file owner/group to unix uid/gid
        // Show as uid==-1 (nobody) if owner does not exist in unix
        // Show as gid==-1 (nogroup) if group does not exist in unix
        stat.st_uid.set(AlluxioFuseUtils.getUid(status.getOwner()));
        stat.st_gid.set(AlluxioFuseUtils.getGidFromGroupName(status.getGroup()));
      } else {
```